app/models/concerns/image_file.rb

```diff
@@ -79,7 +79,7 @@ def url
   end
 
   def href
-    status_complete? ? url : original_url
+    (status_complete? || status_invalid?) ? url : original_url
   end
 
   def href=(h)
```

Comment:
`href=(h)`? What does the `=(h)` syntax do on a method definition?

Reply:
That's how you define a "setter method" in Ruby.  `href` isn't actually the name of any database field.  But you can call both:

```ruby
# returns some string
image.href

# set it to something
image.href = "foo"
```

The first case calls the method up on line 81.

In that 2nd case, it calls this method, with `h` set to the string `"foo"`.